android/core/src/main/java/com/mongodb/stitch/android/core/auth/internal/StitchUserImpl.java

```diff
@@ -32,8 +34,10 @@ public StitchUserImpl(
       final String loggedInProviderName,
       final StitchUserProfileImpl profile,
       final StitchAuthImpl auth,
-      final boolean isLoggedIn) {
-    super(id, deviceId, loggedInProviderType, loggedInProviderName, profile, isLoggedIn);
+      final boolean isLoggedIn,
+      final Date lastAuthActivity) {
+    super(id, deviceId, loggedInProviderType, loggedInProviderName,
```

Comment:
[minor] does this change also need to be made on the server version of this file?